### PR TITLE
Actor invocation perf results.

### PR DESCRIPTION
**Pull request by artursouza**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

**Review by yaron2 — APPROVED** [review]

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Actors activation performance"
+linkTitle: "Actors activation performance"
+weight: 10000
```
Comment (orizohar) [inline daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: for this to go under service invocation please change to "20000"

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+description: ""
+---
+This article provides service invocation API performance benchmarks and resource utilization for Actor in Dapr on Kubernetes.
```
Comment (orizohar) [inline daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: "utilization for an Actor" or "utilization for Actors"

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Placement (required for Actors, control plane mapping ActorType to hosts)
+* Operator (control plane)
+* Sidecar Injector (control plane)
+* Sentry (optional, control plane)
+
+For more information see [overview of Dapr on Kubernetes]({{< ref kubernetes-overview.md >}}).
+
+## Performance summary for Dapr v1.0
+
+The Actors API in Dapr sidecar will identify which hosts are registered for a given Actor type and route the request to the appropriate host for a given Actor Id. The host runs an instance of the application and uses the Dapr SDK (.Net, Java, Python or PHP) to handle Actor requests via HTTP.
```
Comment (orizohar) [inline daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: Id --> ID

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+title: "Actors activation performance"
+linkTitle: "Actors activation performance"
+weight: 10000
+description: ""
+---
+This article provides service invocation API performance benchmarks and resource utilization for Actor in Dapr on Kubernetes. 
+
+## System overview
+
+For applications using Actors in Dapr there are two aspects to be considered. First, is the routing of actor invocations handled by Dapr sidecar. Second, is the Actor runtime that is implemented and handled on the application side and depends on the SDK. For now, the performance tests are using the Java SDK to provide an Actor runtime in the application.
```
Comment (orizohar) [inline daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: lets be consistent with capitalizing (actor or Actor)

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+### Results
+
+* The actual throughput was ~500 qps.
+* The tp90 latency was ~3ms.
+* The tp99 latency was ~6.2ms.
+* Dapr app consumed ~523m CPU and ~304.7Mb of Memory
+* Dapr sidecar consumed 2m CPU and ~18.2Mb of Memory
+* No app restarts
+* No sidecar restarts
```
Comment (orizohar) [inline daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: I recommend adding "Related links" section with links to the actor overview pages (under developing applications / building blocks)

**Review by orizohar — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md (hunk 2 above)
Comment (msfussell) [reply daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: actors - plural

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md (hunk 4 above)
Comment (msfussell) [reply daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: Yes. You should use lowercase "actor" and "actors". This is not a name.

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md (hunk 3 above)
Comment (msfussell) [reply daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: It is hard, but you need to minimize the use of "will" and use present tense as much as possible "will identify which hosts" reword to "The actors API in Dapr sidecar identifies which hosts are registered..." as in this is what it does today.

File: daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+The test was conducted on a 3 node Kubernetes cluster, using commodity hardware running 4 cores and 8GB of RAM, without any network acceleration.
+The setup included a load tester ([Fortio](https://github.com/fortio/fortio)) pod with a Dapr sidecar injected into it that called the service invocation API to reach a pod on a different node.
+
+Test parameters:
+
+* 500 requests per second
+* 1 replica
+* 1 minute duration
+* Sidecar limited to 0.5 vCPU
+* Sidecar mTLS enabled
```
Comment (msfussell) [inline daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md]: mTLS enabled?

**Review by orizohar — APPROVED** [review]

**Pull request merged**